README.md

```diff
@@ -584,9 +600,17 @@ Troubleshooting
 ```
 
  - Snap support is still experimental and something may not work as expected
-   (keep in mind that virtme-ng will try to run snapd in a bare minimum system
-   environment without systemd), if some snaps are not running try to disable
-   apparmor, adding `--append="apparmor=0"` to the virtme-ng command line.
+   (keep in mind that, by default, virtme-ng will try to run snapd in a bare
+   minimum system environment without systemd), if some snaps are not running
+   try to disable apparmor, adding `--append="apparmor=0"` to the virtme-ng
+   command line.
+
+ - Systemd support (`--systemd`) is still experimental, be aware that you might
+   also need `--user root`, or if you're using your own `/` as ROOTFS, you need
+   to run vng itself as root. If something does not work for you, you might
+   need to mask some services. You can see the ones that are masked by default
+   in the CONF_DEFAULT definition, which can be customized via configuration file
+   (refer to the "Default options" section).
```

Comment:
I think it would help if you add an example here, a bit similar to what was done for `default_opts`, because reading the code to know how to write the config file is not nice for the users I think :)

I would prefer to have the default values only in a config file that is shipped with vng, but it doesn't seem to work. We might then have to maintain a copy in the `README` to help users (or a link to the code, but then we will depend on GitHub).

If you do that, please add a comment above the declaration of `CONF_DEFAULT` as a reminder to update the README file if we change it.

Reply:
As a user, I'd prefer to have some proper documentation and leave configuring to the user, so it's best to have the default in the README. Do you agree with this approach?

Reply:
Sorry, end of the week, I don't think I understood your approach. Do you mind rephrase it please? :)

I think the users need some guidance on how to change the configuration. If it cannot be done by simply modifying a local file, it would be good to show how it should look like, no?

Reply:
Added https://github.com/arighi/virtme-ng/pull/280/commits/878eac47344c0cfc78e0476feadc2317ef56e91c, not sure about the wording but that seems to be the way to go IMHO.